README.md

```diff
@@ -68,6 +69,16 @@ Install the build dependencies:
 sudo yum install git gcc gcc-c++ make cmake libarchive-devel libcurl-devel openssl-devel qt5-qtbase-devel qt5-qtquickcontrols2-devel qt5-qtsvg-devel qt5-linguist
 ```
 
+Note that on Centos8 Stream this is slightly different; you need dnf instead of yum, and the p
```

Comment:
Ugh, can you fix the word-wrapping please, so that you don't get spaces in the middle of words!